roadie-rails.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "roadie", ">= 3.1", "< 5.0"
```

Comment:
You should relax 'railties' dependency as well to make it work.

Reply:
Railties is not added explicitly here, though.

Reply:
It is a gem runtime dependency, defined just one line above my comment

Reply:
Ha ha, you are right. I forgot to expand the lines in the diff. Thank you for keeping me on my toes. :man_facepalming: 

I agree then. That too should be adjusted.